card.py

```diff
@@ -1,10 +1,11 @@
 
 
 class Card:
-    def __init__(self, num=None, color=None, is_change_color=False):
+    def __init__(self, num=None, color=None, is_change_color=False, is_stop_card=False):
```

Comment:
The structure can work but starting to have some assumption, like is_change_color and is_stop_card and be true at the same time. basically, the way you really want to model it that each card can have only one type (normal_card., change_color, stop), and now to use two boolean to reflect that can be problematic ( and when you are going to add even more type it going to become unmaintainable)

I would suggest using enum here :) (Ya it is a huge nice refactoring)